src/command/uninstall.rs

```diff
@@ -15,10 +14,8 @@ impl Command for Uninstall {
     fn run(self, session: &mut Session) -> Fallible<ExitCode> {
         session.add_event_start(ActivityKind::Uninstall);
 
-        let version = VersionSpec::default();
-        let tool = tool::Spec::from_str_and_version(&self.tool, version);
-
-        tool.uninstall()?;
+        let tool = tool::Spec::try_from_str(&self.tool)?;
```

Comment:
How about 
```rust
        for tool in Spec::from_strings(&self.tools, "uninstall")? {
            tool.resolve(session)?.uninstall(session)?;
        }
```
just the same way as install which enable to install/uninstall multiple node/package once.